test.sh

```diff
@@ -99,7 +99,7 @@ function tear_down() {
 
 function reset_containers() {
     docker_compose stop -t 0 &>> ${LOGFILE};
-    docker_compose rm -vf &>> ${LOGFILE};
+    docker_compose rm -f &>> ${LOGFILE};
```

Comment:
That's true, and maybe it's even a good idea to leave it in, since it assures a clean state. I will revert this change.